powdr/src/lib.rs

```diff
@@ -1,14 +1,12 @@
 #![no_std]
 #![no_main]
 
+extern crate runtime;
```

Comment:
why this?

Reply:
If I don't make any reference to a dependency, it is not linked.

If `runtime` is not linked, panic handler is not provided and I get a compilation error.

Since I don't use directly any function from `runtime`, instead of `use runtime;` I reference it with `extern crate runtime;` so I don't get a warning of unused import.